plugins/split-yaml.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: split-yaml
+spec:
+  version: v0.1.0
+  homepage: https://github.com/nathforge/kubectl-split-yaml
+  shortDescription: Split Kubernetes YAML output into one file per resource.
+  description: Split Kubernetes YAML output into one file per resource, e.g `kubectl get deploy -o yaml | kubectl split-yaml -p deployments`
```

Comment:
Could you wrap that line at ~80 chars, so that it prints nicely on wide and narrow terminals?

Reply:
Sure, that's done.